general-concepts/licenses/text.xml

```diff
@@ -39,6 +37,46 @@ be used) then use the following syntax:
 LICENSE="|| ( foo bar )"
 </codesample>
 
+<section>
+<title>License-implied restrictions</title>
+<body>
+
+<p>
+Non-free licenses may require additional restrictions to be stated
+in the ebuild. In order to establish the correct restrictions, it is
+necessary to read the license and establish the applicable clauses
+based on files contained in upstream packages (upstreams may use
+the same license for multiple products, with some restrictions not being
+applicable to the ebuild in question).
+</p>
+
+<p>
+If the license of a package does not explicitly permit redistributing
+distfiles found in <c>SRC_URI</c>, the corresponding ebuilds must have
+<c>RESTRICT=mirror</c> to prevent the file from being copied to Gentoo
```

Comment:
```suggestion
<c>RESTRICT=mirror</c> to prevent the affected files from being copied to Gentoo
```

Reply:
Technically, RESTRICT allows use-conditional dependency specifications right? `RESTRICT="foo? ( mirror )` should be valid assuming `USE=foo` pulls in a mirror-restricted distfile. I just want to establish that RESTRICT is not an all-or-nothing approach for ebuilds with multiple distfiles.

Reply:
Yes.